assets/scripts/scripts.js

```diff
@@ -923,4 +986,20 @@ async function decoratePage(doc) {
   }, 3 * 1000);
 }
 
-decoratePage(document);
+function init() {
+  sampleRUM('top');
+
+  window.addEventListener('load', () => sampleRUM('load'));
+
+  window.addEventListener('unhandledrejection', (event) => {
+    sampleRUM('error', { source: event.reason.sourceURL, target: event.reason.line });
+  });
+
+  window.addEventListener('error', (event) => {
+    sampleRUM('error', { source: event.filename, target: event.lineno });
+  });
```

Comment:
I like to keep this just below the `sampleRUM` definition, so that the RUM code isn't as scattered.